src/flicking.js

```diff
@@ -175,7 +176,7 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		_build: function () {
 			var panel = this._conf.panel;

```

Comment:
Is this line passed convention? for... coding convention such as "variable name of DOM elements should have '$' prefix".